awx/main/expect/isolated_manager.py

```diff
@@ -241,15 +245,20 @@ def check(self, interval=None):
 
             # discover new events and ingest them
             events_path = self.path_to('artifacts', self.ident, 'job_events')
-            for event in set(os.listdir(events_path)) - handled_events:
-                path = os.path.join(events_path, event)
-                if os.path.exists(path):
-                    event_data = json.load(
-                        open(os.path.join(events_path, event), 'r')
-                    )
-                    event_data.setdefault(self.event_data_key, self.instance.id)
-                    dispatcher.dispatch(event_data)
-                    handled_events.add(event)
+
+            # it's possible that `events_path` doesn't exist *yet*, because runner
```

Comment:
@chrismeyersfsu @Spredzy this is another bug I noticed while testing, and I think it explains some of the weird errors we've misinterpreted as pathing errors:

![image](https://user-images.githubusercontent.com/214912/54956385-3bac2600-4f26-11e9-89f9-ff90ae24b4a1.png)

It doesn't look like runner actually creates this directory until it rights its first events, so I found that it was possible to hit a race condition where we launched runner and then checked quickly (and this directory didn't exist yet in the rsync that came back).